src/client/app/containers/LineChartContainer.js

```diff
@@ -7,24 +7,24 @@
 import { Line } from 'react-chartjs-2';
 import moment from 'moment';
 import { connect } from 'react-redux';
-import GraphColors from '../utils/GraphColors';
+import getGraphColor from '../utils/getGraphColor';
 
 /**
  * @param {State} state
  */
 function mapStateToProps(state) {
 	const timeInterval = state.graph.timeInterval;
 	const data = { datasets: [] };
-	const graphColors = new GraphColors();
 
 	for (const meterID of state.graph.selectedMeters) {
 		const readingsData = state.readings.line.byMeterID[meterID][timeInterval];
 		if (readingsData !== undefined && !readingsData.isFetching) {
+			const label = state.meters.byMeterID[meterID].name;
 			data.datasets.push({
-				label: state.meters.byMeterID[meterID].name,
+				label,
```

Comment:
I personally don't like inconsistent use of insert-by-name syntax in object literals, but feel free to keep it if that's the project style.

Reply:
We've been using this everywhere.